src/components/Body/NftList/NftList.js

```diff
@@ -19,6 +20,40 @@ const contract = getContract({
 
 const NftList = () => {
   const [nftList, _setNftList] = useState([]);
+  const [filteredNftList, _setFilteredNftList] = useState(false);
+  const [searchQuery, _setSearchQuery] = useState('');
+
+  const onSearchBtnClick = () => {
+    try {
+      if (searchQuery?.length) {
+        const trimmedSearchVal = searchQuery.trim();
+    
+        const filteredResults = nftList.filter((nft) => {
+          const githubId = nft.attributes[1].value;
+    
+          return githubId.includes(trimmedSearchVal);
```

Comment:
Hey @saidarshan27  you did a good work. Let's refine your contribution a bit more.

## Contribution Request: Remove Case Sensitivity in Search Bar Input
- [ ] We would like to improve the search functionality by removing case sensitivity. The goal is to ensure that the search works regardless of whether users input uppercase or lowercase letters. Please modify the search logic to handle all inputs uniformly by converting both the search query and the searchable data to the same case (either lowercase or uppercase) before performing the search.

- [ ] Before submitting your pull request, please ensure that your forked repository is in sync with the latest code from the main branch of this repository. Here are the steps to do so:

Reply:
Hey @bhupendra-chouhan ,

I have made the search functionality to be case insensitive. Also I have added a Loader component.

The loader component accepts a type prop: 'inline','partial','full'
As the names suggest, inline type loader can be used inside buttons.
partial is not a full screen loader without the background.
full is full screen loader, which will overlay the entire screen.

You can also control the loader color as well as the background color in `loader.css`